packages/cli/src/commands/set-version.ts

```diff
@@ -0,0 +1,169 @@
+import chalk from "chalk";
+import { existsSync, readFileSync, rmSync, writeFileSync } from "fs";
+import path from "path";
+import type { CommandModule } from "yargs";
+import { logError, MUDError } from "../utils/errors.js";
+
+type Options = {
+  backup?: boolean;
+  force?: boolean;
+  restore?: boolean;
+  mudVersion?: string;
+};
+
+const BACKUP_FILE = ".mudbackup";
+
+const commandModule: CommandModule<Options, Options> = {
+  command: "set-version",
+
+  describe: "Install a custom MUD version and optionally backup the previously installed version",
+
+  builder(yargs) {
+    return yargs.options({
+      backup: { type: "boolean", description: "Back up the current MUD versions to `.mudinstall`" },
```

Comment:
should this be `.mudbackup` (value of `BACKUP_FILE` variable above)?

Reply:
indeed, will use the variable